### PR TITLE
Check all ⚡️emojis in the twitter bio

### PR DESCRIPTION
**Pull request by bumi**
So far we had only looked for the first occurance of the ⚡️emoji. This means we did not find any lightning address when there have been multiple emojis before the ln address.
This change iterates through all or the elements and tries to find an ln address.

### CODE REVIEW

**Review by dylancom — CHANGES_REQUESTED** [review]

File: src/extension/content-script/batteries/Twitter.ts [hunk 1]
```diff
@@ -99,13 +99,14 @@ function battery(): void {
         //  );
         //});
         // if we find a ⚡ emoji we use the text of the next sibling and try to extract a lnurl
-        if (zapElement) {
+        for (let i = 0; i < zapElements.length; i++) {
```
[the remark this answers is not in the document]
Comment (bumi) [reply src/extension/content-script/batteries/Twitter.ts:102]: I need to break early, so a forEach would not work would it?

[the remark this answers is not in the document]
Comment (dylancom) [reply src/extension/content-script/batteries/Twitter.ts:102]: Nope, so we can use the modern for of.

**Review by dylancom — APPROVED** [review]

File: src/extension/content-script/batteries/Twitter.ts [hunk 2]
```diff
@@ -99,13 +99,14 @@ function battery(): void {
         //  );
         //});
         // if we find a ⚡ emoji we use the text of the next sibling and try to extract a lnurl
-        if (zapElement) {
+        for (const zapElement of Array.from(zapElements)) {
```
Comment (dylancom) [inline src/extension/content-script/batteries/Twitter.ts:102]: Array.from is not needed if we set the target in our tsconfig.json to ES6: `"target": "ES6"`.
This is supported in all modern browsers.

**Review by dylancom — CHANGES_REQUESTED** [review]

File: src/extension/content-script/batteries/Twitter.ts (hunk 2 above)
Comment (bumi) [reply src/extension/content-script/batteries/Twitter.ts:102]: let's do that

**Review by dylancom — APPROVED** [review]

**Pull request merged**